### PR TITLE
Added Publications 

### PR DESCRIPTION
**Pull request by saadaziz33**
publications, live coding menu entry added
change in color of sub headings on home page

### CODE REVIEW

**Comment by rmgk** [comment]
please separate the changes to the publications into a single commit without the other unrelated files, and without the intermediate commits.

File: Pubications.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: Publications
+nav: 6
+---
+
+# Publications
+
+
+G. Salvaneschi, S. Proksch, S. Amann, S. Nadi, M. Mezini, On the Positive Effect of Reactive Programming on Software Comprehension: An Empirical Study", IEEE Transactions on Software Engineering, 2017
```
Comment (rmgk) [inline Pubications.md]: no links yet

File: Pubications.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+Guido salvaneschi, What do we Really Know about Data Flow Languages, PLATEAU 2016
+
+Guido Salvaneschi, Mira Mezini, Debugging Reactive Programming with Reactive Inspector, ICSE 2016
+
+<a href="https://saadaziz33.github.io/REScala/2016_Debugging-for-Reactive-Programming_pdf.pdf" type="application/pdf" width="100%" height="100%">Guido Salvaneschi, Mira Mezini, "Debugging for Reactive Programming", ICSE 2016</a>
```
Comment (rmgk) [inline Pubications.md]: links to pdfs

File: .idea/vcs.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (rmgk) [inline .idea/vcs.xml]: remove all the extra files

**Comment by saadaziz33** [comment]
How about now?

File: Pubications.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+---
+title: Publications
+nav: 6
+---
+
+# Publications
+
+G. Salvaneschi, S. Proksch, S. Amann, S. Nadi, M. Mezini, [_On the Positive Effect of Reactive Programming on Software Comprehension: An Empirical Study_](http://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=7827078), IEEE Transactions on Software Engineering, 2017<br>
+Guido salvaneschi, [_What do we Really Know about Data Flow Languages_](https://dl.acm.org/ft_gateway.cfm?id=3001884&ftid=1805603&dwn=1&CFID=4795785&CFTOKEN=eb0c0db87e73e7dc-A7B5A032-0B68-D48C-0891DDB0F19390E6), PLATEAU 2016<br>
+Guido Salvaneschi, Mira Mezini, [_Debugging Reactive Programming with Reactive Inspector_](https://dl.acm.org/ft_gateway.cfm?id=2893174&ftid=1722250&dwn=1&CFID=4795785&CFTOKEN=eb0c0db87e73e7dc-A7B5A032-0B68-D48C-0891DDB0F19390E6), ICSE 2016<br>
```
Comment (rmgk) [inline Pubications.md]: the direct links to papers are not valid permanently or outside of the university, we need to link to the official frontpage or some puplic versions instead.
Lets see what Guido thinks.

File: Pubications.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+Guido Salvaneschi, Mira Mezini, [_Debugging for Reactive Programming_](https://saadaziz33.github.io/REScala/2016_Debugging-for-Reactive-Programming_pdf.pdf), ICSE 2016<br>
+Oliver Bracevac, Sebastian Erdweg, Guido Salvaneschi, and Mira Mezini, [_CPL: A Core Language for Cloud Computing_](https://dl.acm.org/ft_gateway.cfm?id=2889452&ftid=1700146&dwn=1&CFID=4795785&CFTOKEN=eb0c0db87e73e7dc-A7B5A032-0B68-D48C-0891DDB0F19390E6), MODULARITY 2016<br>
+[_Reactive Programming: a Walkthrough_](https://dl.acm.org/ft_gateway.cfm?id=2819226&ftid=1617074&dwn=1&CFID=4795785&CFTOKEN=eb0c0db87e73e7dc-A7B5A032-0B68-D48C-0891DDB0F19390E6) G. Salvaneschi, A. Margara, G. Tamburrelli. ICSE Technical Briefing. In Proceedings of the 37th International Conference on Software Engineering (ICSE '15). Florence, Italy. May 16 - 24, 2015.<br>
```
Comment (rmgk) [inline Pubications.md]: remove the <br> tags and use either markdown paragraphs (two newlines) or lists.

**Comment by saadaziz33** [comment]
There are 3 pdf files, 1 publications.md, 1 config.yml, 1 studies.md (no idea where is this from :/)

File: _config.yml [hunk 6]
```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-cayman
```
Comment (rmgk) [inline _config.yml]: still unrelated changes in the pull reuqest

File: studies.md [hunk 7]
```diff
@@ -5,14 +5,12 @@ title: Case Studies
...
+ReactShapes is a small drawing program. The user can drag and drop
 different shapes on a canvas, connect them with
 lines and change the stroke width and the color of
 each shape.
-
-The application supports an history and an undo
-function. Finally, the drawing canvas can be shared with
+The application supports a history with undo.
+Finally, the drawing canvas can be shared with
 other clients that participate in the same task from
```
Comment (rmgk) [inline studies.md]: more unrelated changes

File: Pubications.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
+---
+title: Publications
+nav: 6
+---
+
+# Publications
+
+* G. Salvaneschi, S. Proksch, S. Amann, S. Nadi, M. Mezini, [_On the Positive Effect of Reactive Programming on Software Comprehension: An Empirical Study_](http://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=7827078), IEEE Transactions on Software Engineering, 2017
+* Guido salvaneschi, [_What do we Really Know about Data Flow Languages_](https://dl.acm.org/ft_gateway.cfm?id=3001884&ftid=1805603&dwn=1&CFID=4795785&CFTOKEN=eb0c0db87e73e7dc-A7B5A032-0B68-D48C-0891DDB0F19390E6), PLATEAU 2016
+* Guido Salvaneschi, Mira Mezini, [_Debugging Reactive Programming with Reactive Inspector_](https://dl.acm.org/ft_gateway.cfm?id=2893174&ftid=1722250&dwn=1&CFID=4795785&CFTOKEN=eb0c0db87e73e7dc-A7B5A032-0B68-D48C-0891DDB0F19390E6), ICSE 2016
```
Comment (rmgk) [inline Pubications.md]: as discussed with guido, please link to the page of the paper instead of to the pdf directly, as the pdf link is not the canonical link

**Comment by saadaziz33** [comment]
- links updates, 
- intermediate commits excluded
- unrelated files reverted

**Comment by rmgk** [comment]
Final result looks good. However, if you squash them into a single one (containing only the final results) then this should merge cleanly.

**Comment by saadaziz33** [comment]
commits squashed.
Actually fixup as we don't need the commit messages.
